lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb

```diff
@@ -421,4 +421,91 @@ def model_apply_prm_baseline_skylight_to_roof_ratio(model)
 
     return true
   end
+
```

Comment:
We might want to add the YARD comments here before merging.

Reply:
done